v5/v5_stable.ino

```diff
@@ -16,7 +16,9 @@ String whatsapp_numbers[4] = {WHATSAPP_NUMBER_1, WHATSAPP_NUMBER_2,
 String API[4] = {WHATSAPP_API_1, WHATSAPP_API_2, WHATSAPP_API_3,
                  WHATSAPP_API_4};
 
-// https://api.callmebot.com/whatsapp.php?phone=+923354888420&text=This+is+a+test&apikey=518125
+const char *mqtt_server = MY_MQTT_SERVER_IP;
+
+// https://api.callmebot.com/whatsapp.php?phone=+923354***420&text=This+is+a+test&apikey=***125
```

Comment:
Instead of hiding whatsapp API I suggest you to move to to any git-ignore file so you can use it easily.

Reply:
added